checkPropTypes.js

```diff
@@ -39,7 +39,9 @@ if (process.env.NODE_ENV !== 'production') {
  * @param {?Function} getStack Returns the component stack.
  * @private
  */
-function checkPropTypes(typeSpecs, values, location, componentName, getStack) {
+function checkPropTypes(typeSpecs, values, location, componentName, getStack, warningLogger) {
+  var warningLogger = warningLogger || null;
```

Comment:
this line isn't necessary; if it's falsy, it will skip the `if` branch below already.
```suggestion
```

Reply:
Fixed